widgets/executions/src/tasksGraph/ExecutionWorkflowGraph.tsx

```diff
@@ -41,9 +41,12 @@ export default function ExecutionWorkflowGraph({ containerHeight, selectedExecut
 
     const timer = useRef(null);
     const cancelablePromise = useRef(null);
-    const wrapper = useRef();
     const modal = useRef();
 
+    const wrapper = useRef();
+    const [wrapperWidth, setWrapperWidth] = useState();
```

Comment:
Minor:

Since this is a TS file, consider writing valid TS to make the migration easier :slightly_smiling_face: 

```suggestion
    const [wrapperWidth, setWrapperWidth] = useState<number>(null);
```

or something else :smile: 

Same for the `wrapper` above (probably we want to use `HTMLElement`)

Reply:
Done